apiserver/facadeversions_test.go

```diff
@@ -21,6 +21,7 @@ type facadeVersionSuite struct {
 var _ = gc.Suite(&facadeVersionSuite{})
 
 func (s *facadeVersionSuite) TestFacadeVersionsMatchServerVersions(c *gc.C) {
+	c.Skip("Skip until the resourceshookcontext and resource facades are re-enabled.")
```

Comment:
**question:** Are those line to restore referenced in some JIRA ? Could be good to:

* Keep links somewhere
OR
* add a `// TODO(dqlite/wrapup/resources): a cool explanation`

To avoid to forget some of them

Reply:
the Skip implies the todo

Added to JUJU-7066.